scripts/v6/asset-fin-forms.sql

```diff
@@ -0,0 +1,498 @@
+[RiskExposure], [ForecastReplYear], [ForecastReplValue], [CRC_Adjusted], [DRC], [AnnualisedMaintPct], [MaintenanceBudgetNeed], [MaintenanceBudgetPct], [MaintenanceExpenditure], [DateCreated], [DateLastFinMonth],
+[ImpairmentAll], [RevImpairmentAll], [Asset_Barcode_Nr], [Room_Barcode_Nr], [Room_Number], [Building_Name], [Serial_Number], [Fleet_Number], [Fleet_Reg_Year], [Treasury_Code], [Cost_Centre], [Acquisition_Date],
+[Impairment_Test_Date], [Impairment_Review_Date], [EUL_Review_Date], [RV_Review_Date], [Depr_Review_Date], [Condition_Review_Date], [Resp_CostCentre_ID], [Maint_CostCentre_ID], [PLANT_ID], [WorkCentre_ID], [LOC_ID],
+[CriticalityGrade_CG], [ConditionGrade_CG], [PerformanceGrade_CG], [UtilisationGrade_CG], [OpsCostGrade_CG], [YearConstruct_CG], [YearRenewed_CG], [ImpairmentDerecog], [FairValueLessCostSell], [ValueInUse], [ImpairmentDate],
+[RevImpairmentDate], [ImpairmentReason], [RevImpairmentReason], [TransferCost], [TransferDepr], [TransferDate], [RevaluationReserveOpening], [RevaluationReserveFinYTD], [RevaluationReserveClosing], [RefSuburbsCode],
+[CaseNumber], [InsuranceClaimed], [InsuranceAmount], [TransferredFrom], [TransferredTo], [EquipmentNr], [ImpairmentTransfer],
+[SCOA_Fund], [SCOA_Function], [SCOA_Mun_Classification], [SCOA_Project], [SCOA_Costing], [SCOA_Region], [SCOA_ItemAsset], [FinYear])
+VALUES (
+	N'P000000001', 1, N'INF_GEN1_WAS60019_ROA1_EAR.4', N'000000989069', N'IMM', N'PPE', N'INF', N'WWTW', N'BAL', N'RDF', N'BLBD', NULL, NULL, N'Test Project', N'', NULL, NULL, N'', N'test_comp1',
+	CAST(10.00 AS Numeric(18, 2)), N'EA', CAST(37730.00 AS Numeric(18, 2)), NULL, NULL, 'Dummy - Region 01', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, CAST(N'2013-11-15 00:00:00.000' AS DateTime),
+	NULL, NULL, NULL, N'TestProject001', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 2013, NULL, NULL, NULL, CAST(25000.00 AS Numeric(18, 2)),
+	NULL, NULL, NULL, NULL, 15, NULL, CAST(15.00 AS Numeric(18, 2)), 0, CAST(0.00 AS Numeric(18, 2)), N'ZJW1', NULL, CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, CAST(377300.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, CAST(377300.00 AS Numeric(18, 2)), NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, N'comp1', NULL, NULL, NULL, N'78987', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 78987, 78987, 78987, 1, 99092, NULL, NULL, NULL, NULL, NULL,
+	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, CAST(0.00 AS Numeric(18, 2)), CAST(0.00 AS Numeric(18, 2)), CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL,
+	N'240612638294574528', CAST(0.00 AS Numeric(18, 2)), NULL, NULL, NULL, NULL, NULL, NULL, NULL, 2015
+);
+INSERT [dbo].[AssetFinFormInput](
```

Comment:
We could also refactor these insert statements....get rid of NULL values that we don't really need them.